Mathlib/Topology/Separation/Hausdorff.lean

```diff
@@ -674,12 +674,21 @@ theorem IsQuotientMap.of_surjective_continuous [CompactSpace X] [T2Space Y] {f :
 @[deprecated (since := "2024-10-22")]
 alias QuotientMap.of_surjective_continuous := IsQuotientMap.of_surjective_continuous
 
+theorem isPreirreducible_iff_forall_subset_closure_singleton [R1Space X] {S : Set X} :
+    IsPreirreducible S ↔ ∀ x ∈ S, S ⊆ closure {x} := by
+  constructor
+  · intro h x hx y hy
+    by_contra e
+    obtain ⟨U, V, hU, hV, hxU, hyV, h'⟩ := r1_separation fun h => e h.specializes.mem_closure
```

Comment:
```suggestion
    obtain ⟨U, V, hU, hV, hxU, hyV, h'⟩ := r1_separation fun h ↦ e h.specializes.mem_closure
```

Reply:
This is not the preferred mathlib convention, see
https://leanprover-community.github.io/contribute/style.html#anonymous-functions

Can I suggest that you do not resolve conversation if there is a conversation going on? We normally do this once the author tacitly accepts a reviewer's suggestion, but I believe it is nicer to keep it open when we disagree with the reviewer.